packages/lit-element/src/lib/platform-support.ts

```diff
@@ -0,0 +1,377 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+/**
+ * LitElement/lit-html patch to support browsers without native web components.
+ *
+ * @packageDocumentation
```

Comment:
I _think_ this should be `@file`. Not 100% sure though.

Reply:
Let's investigate separately and change in all locations if so.